velox/docs/functions/uuid.rst

```diff
@@ -0,0 +1,7 @@
+=============================
+UUID Functions
+=============================
+
+.. function:: uuid() -> varchar
+
+   Returns a pseudo randomly generated UUID as a single character string
```

Comment:
The signature and semantics must match Presto SQL exactly, hence, it needs to return a value of type UUID, not VARCHAR.